src/storage/types.rs

```diff
@@ -69,10 +69,12 @@ impl Mutation {
 /// Represents the status of a transaction.
 #[derive(PartialEq, Debug)]
 pub enum TxnStatus {
-    /// The txn is just rolled back by the current command.
-    Rollbacked,
     /// The txn was already rolled back before.
     RollbackedBefore,
```

Comment:
How about just name this `Rollbacked` here? When I wrote this I can't come up with a better name, and have to distinguish it from the old one `Rollbacked`, so I named it `RollbackedBefore`.

Reply:
Good idea.